lib/TileBag.rb

```diff
@@ -0,0 +1,34 @@
+module Scrabble
+DEFAULT_TILES = {A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2, I: 9, J: 1, K: 1, L: 4, M: 2, N: 6, O: 8, P: 2, Q: 1, R: 6, S: 4, T: 6, U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1}
+  class TileBag
+   attr_reader :tile_quantities
+    def initialize
+      @tile_quantities = DEFAULT_TILES.dup
+    end
+    def draw_tiles(num)
```

Comment:
This whole `draw_tiles` method might be a lot simpler if we used the `DEFAULT_TILES` constant as a hash but inside of `initialize` we created an array from it with multiple entries for each letter that is duplicated:
```ruby
def initialize
  @tiles = []
  DEFAULT_TILES.each do |letter, quantity|
    quantity.times { @tiles.push(letter) }
  end
end
```